README.md

```diff
@@ -1,5 +1,4 @@
-<p align="center"><h1>Hyperledger URSA</h1></p><br>
----
+<p align="center"><h1>Hyperledger URSA</h1></p><br> 
```

Comment:
It seems weird to have HTML in a MD file--I suggest just `# Hyperledger URSA` 
The centering is ignored anyway.

Reply:
Fixed